MAUI/Scheduler/reminders.md

```diff
@@ -0,0 +1,423 @@
+---
+layout: post
+title: Appointments Reminders in .NET MAUI Scheduler control | Syncfusion
+description: Learn here all about the appointments reminders support in Syncfusion .NET MAUI Scheduler (SfScheduler) appointments.
+platform: maui
+control: SfScheduler
+documentation: ug
+---
+
+# Appointments reminders in .NET MAUI Scheduler (SfScheduler)
+
+The MAUI Scheduler notify an appointment reminders by using  the [EnableReminder] property and [ReminderAlertOpening] event. An appointment can have one or more reminders.
+
+N>
+* As restriction in enabling toast notification in .NET MAUI Framework, as of now added event to notify appointment reminder.[Reference](https://github.com/dotnet/maui/discussions/4216)
+
+## Handling reminders for future appointment
+
+* Future normal and recurring appointment reminder alert will be displayed as Reminder alert time.
+
+* Reminder alert time will be calculated form appointment start time substrate with reminder time interval. For an example if appointment time is 3.30pm and reminder time interval is new Timespan (0,15,0) then reminder alter time is 3.15 pm.
+
+## Handling reminders for overdue appointment
+
+* If a Non-recurring appointments reminder is in past, reminder alert as overdue untill dismissed.
+
+* Recurring appointment overdue after recurrence end until dismissed and past occurrences do not have reminders.
```

Comment:
and past occurrences do not have reminders. - what are you conveying here?

Reply:
Changed as - Recurring appointment overdue from the last occurrence date and time until the last occurrence is dismissed.